app/src/main/java/org/xbmc/kore/ui/MoviesActivity.java

```diff
@@ -67,10 +80,15 @@ protected void onCreate(Bundle savedInstanceState) {
 
             // Setup animations
             if (Utils.isLollipopOrLater()) {
-                movieListFragment.setExitTransition(null);
+                //Fade added to prevent shared element from disappearing very shortly at the start of the transition.
+                Fade fade = new Fade();
+                fade.setDuration(50);
+                movieListFragment.setExitTransition(fade);
```

Comment:
Can't the default android.R.transition.fade be used? It's used in the reenter transition (line bellow), so it could be inflated once, and be used in these 2 places.